### DISCUSSION TITLE
Power Strips

### DISCUSSION THREAD

**Discussion by HarryC1998** [netbox-community/netbox#10354] [General]
Hi All, 

I have 42U rack with power strips going down the sides of the rack. 

I can't see an easy way of connecting a PDU to the back whilst utilising the space for the servers too.

**Comment by jeremystretch** [comment]
These are what we call "0U" devices: They can be associated with a rack but not with any particular unit(s) within the rack, and they don't count as consuming rack space. Just assign these devices to the rack without specifying a position or rack face and you're good to go.

**Reply by HarryC1998** [reply]
Hi Jeremy, 

That makes sense. Thanks for your help.
